File: Python Programming/problems on bitwise operator/problem18.py
'''
17. Write a program which accept one number from user and check whether 9th or 12th bit is on or off. 
	
	Input :  257  
	Output :  TRUE
	
	Author : Milind Nalawade
	
	Date : 04/08/2021
	
	Language : Python 3.9.0

'''

import sys
#====
#Logic
#====
def ChkBit(no,ino1,ino2):
	i=1<<(ino1)
	j=1<<(ino2)
	if no&i>0 or no&j>0:
		return 1
	else:
		return 0
		
#=============
#Entry point function
#=============
def main():
	no=int(sys.argv[1])
	ino1=int(sys.argv[2])
	ino2=int(sys.argv[3])
	ret=ChkBit(no,ino1,ino2)
	if ret==1:
		print("\n TRUE \n")
	else:
		print("\n FALSE \n")
		
#=====
#starter
#=====
if __name__=="__main__":
	main()

File: C++ Programming/problems on bitwise operator/problem18.cpp
/*
18. Write a program which accept one number , two positions from user and check whether bit at first or bit at second position is ON or OFF. 
	
	Input :  10 3 7 
	Output :  TRUE 
	
	Author : Milind Nalawade
	
	Date : 04/08/2021
	
	Language : C programming

*/
using namespace std;

#include<iostream>
#include<stdlib.h>

int ChkBit(int no,int first,int last)
{
	int i=1<<(first);
	int j=1<<(last);
	if(no&i||no&j)
	{
		return 1;
	}
	else
	{
		return 0;
	}
}

int main(int argc,char* argv[] )
{
	int no=atoi(argv[1]);
	int first=atoi(argv[2]);
	int last=atoi(argv[3]);
	int ret=0;
	ret=ChkBit(no,first,last);
	if(ret==1)
	{
		cout<<"\n TRUE \n";
	}
	else
	{
		cout<<"\n FALSE \n";
	}
	return 0;
}